packages/firestore/src/platform/browser/dom.ts

```diff
@@ -28,3 +28,17 @@ export function getDocument(): Document | null {
   // eslint-disable-next-line no-restricted-globals
   return typeof document !== 'undefined' ? document : null;
 }
+
+/**
+ * An instance of the Platform's 'TextEncoder' implementation.
+ */
+export function newTextEncoder(): TextEncoder {
```

Comment:
This should only be in platform is there is some divergence between platforms. This doesn't seem to be the case?

Reply:
Sad :/ 

Should we move this to the ./serializer.ts file though? It seems much more fitting.

Reply:
Hmm, but it is not used in `serializer.ts`. The only place it is used in non-testing code is `util/bundle_reader.ts`, I think it makes slightly more sense to keep it here.

Reply:
I mean to move it from packages/firestore/src/platform/browser/dom.ts to packages/firestore/src/platform/browser/serializer.ts. It has little to do with DOM, mostly because it exists in Node which doesn't have DOM.

Reply:
Ah, sorry. Done.